code/_onclick/click.dm

```diff
@@ -338,27 +338,22 @@
 		facedir(direction)
 
 /obj/screen/click_catcher
+	name = "Darkness"
 	icon = 'icons/mob/screen_gen.dmi'
 	icon_state = "click_catcher"
 	plane = CLICKCATCHER_PLANE
+	layer = LAYER_HUD_UNDER
 	mouse_opacity = 2
-	screen_loc = "CENTER-7,CENTER-7"
-
-/obj/screen/click_catcher/proc/MakeGreed()
-	. = list()
-	for(var/i = 0, i<15, i++)
-		for(var/j = 0, j<15, j++)
-			var/obj/screen/click_catcher/CC = new()
-			CC.screen_loc = "NORTH-[i],EAST-[j]"
-			. += CC
+	screen_loc = "1,1 to 15,15"
```

Comment:
It might be better to use `screen_loc = "SOUTHWEST to NORTHEAST"`, since then it will scale automatically if the client can see more than 15x15 tiles. iirc that comes into play with scopes/binoculars and an admin button.

The `"SOUTHWEST to NORTHEAST"` thing was used to make goggle vision scale automatically in #6809 if you want to see an example.

Reply:
I originally changed that when I found that screenloc2turf couldn't parse that syntax, but seeing as I later refactored it to be a single catcher rather than one for each tile of the viewport, I suppose that concern is nullified.